src/main/java/org/opensearch/knn/index/engine/QueryContext.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.engine;
```

Comment:
create another package `engine.model` and have all the request pojos in there

Reply:
sure